setup.cfg

```diff
@@ -6,12 +6,16 @@ norecursedirs =
 python_files =
     test_*.py
 # doctest_plus = disabled
-addopts = --strict
+addopts =
+    --strict
+    --doctest-modules
+    --durations=0
```

Comment:
does it mean these options are added to every py.test command? 
what does durations=0 mean?

Reply:
yes, it does